java/src/jmri/jmrit/display/layoutEditor/MultiSensorIconFrame.java

```diff
@@ -44,12 +45,15 @@
         addHelpMenu("package.jmri.jmrit.display.MultiSensorIconFrame", true);
     }
 
+    int isEmpty = 0; // check for empty Fields in panel
+    int _numberOfPositions = 3; // add an index to Sensor label
+
     public void initComponents() {
         this.getContentPane().setLayout(new BoxLayout(this.getContentPane(), BoxLayout.Y_AXIS));
 
         JPanel p = new JPanel();
         p.setLayout(new BoxLayout(p, BoxLayout.X_AXIS));
-        p.add(new JLabel("Icon checks click: "));
+        p.add(new JLabel(Bundle.getMessage("IconChecksClickLabel", ":")));
```

Comment:
This pattern is wrong. The formatter property string needs to be a grammatically complete statement, or a string that is complete within the dictates of the UI component (for example, a label that includes a keyboard nemonic as underlined in a Germanic or Romance language, would need to include that same nemonic in parenthesis in Japanese).